hermes_eea/calibration/calibration.py

```diff
@@ -218,7 +272,7 @@ def get_calibration_file(data_filename: Path, time=None) -> Path:
     Examples
     --------
     """
-    return None
+    return os.path.join(hermes_eea._data_directory, data_filename)
```

Comment:
Won't be EEA calibration files likely change for different times? The point of this function is to return the correct calibration file for a specific time and not a fixed file path.

Reply:
Yes, multiple stepper tables haven't been worked out yet.